src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1768,7 +1768,8 @@ private void BindArgumentsAndNames(BracketedArgumentListSyntax argumentListOpt,
         {
             // The old native compiler ignores ref/out in a delegate creation expression.
             // For compatibility we implement the same bug except in strict mode.
-            RefKind refKind = (!isDelegateCreation || Compilation.FeatureStrictEnabled)
+            // Note: An inline delegate should still be rejected
+            RefKind refKind = (!isDelegateCreation || Compilation.FeatureStrictEnabled || (isDelegateCreation && argumentSyntax.Expression.IsAnonymousFunction() ))
```

Comment:
You don't have to break to a new line, your choice. Please remove a space between parenthesis at the end ```) ))```.